GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/UI/StatusBarItem.m

```diff
@@ -51,10 +51,14 @@ - (IBAction) HandleMenuClick:(id) sender
 
 - (IBAction) DisplayAboutBox
 {
-    AboutWindowController *aboutWindowController = [[AboutWindowController alloc]
-        initWithWindowNibName:@"AboutWindowController"];
-    [aboutWindowController showWindow:self];
-    [aboutWindowController.window makeKeyAndOrderFront:self];
+    [[NSApplication sharedApplication] activateIgnoringOtherApps:TRUE];
```

Comment:
This helps prevent our About box from getting hidden behind windows of any currently active app (the one whose menubar and windows are visible).

Reply:
This is needed only because our app will be running in the background almost all the time. An alternate option would be to make the About box a floating window.